backend/src/main/java/com/woowacourse/gongcheck/domain/member/Member.java

```diff
@@ -42,6 +43,16 @@ public Member(Long id, String spacePassword, LocalDateTime createdAt, LocalDateT
         this.updatedAt = updatedAt;
     }
 
+    public void checkPassword(String password) {
+        if (!isSameSpacePassword(password)) {
+            throw new UnauthorizedException("공간 비밀번호와 입력하신 비밀번호가 일치하지 않습니다.");
+        }
+    }
+
+    private boolean isSameSpacePassword(String password) {
+        return this.spacePassword.equals(password);
```

Comment:
```suggestion
        return spacePassword.equals(password);
```
중복된 요소가 아니라면 `this` 키워드를 없애도 되었을 것 같네요 (매개변수를 spacePasword로 받으면 그대로 가지만요 :))